### ISSUE TITLE
Support tmux 3.0 and more

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#141]
https://github.com/tmux/tmux/releases/tag/3.0a

**Comment by greymd** [comment]
In tmux 3.0, REG_STARTEND seems not portable.
Let me skip this version because it cannot be built on the CI env.

```
regsub.c: In function 'regsub':
regsub.c:83:39: error: 'REG_STARTEND' undeclared (first use in this function); did you mean 'TTY_STARTED'?
   83 |   if (regexec(&r, text, nitems(m), m, REG_STARTEND) != 0) {
      |                                       ^~~~~~~~~~~~
      |                                       TTY_STARTED
regsub.c:83:39: note: each undeclared identifier is reported only once for each function it appears in
```

**Comment by greymd** [comment]
Released [v4.1.2](https://github.com/greymd/tmux-xpanes/releases/tag/v4.1.2)
